cmake/templates/cvconfig.h.in

```diff
@@ -40,6 +40,9 @@
 /* NVIDIA CUDA Fast Fourier Transform (FFT) API*/
 #cmakedefine HAVE_CUFFT
 
+/* Default backend used by the DNN module */
+#define OPENCV_DNN_BACKEND_DEFAULT ${OPENCV_DNN_BACKEND_DEFAULT}
```

Comment:
Please use this approach instead: https://github.com/opencv/opencv/blob/4.8.0/modules/core/CMakeLists.txt#L94

Reply:
Alright, fixed in new commit.